lib/http.js

```diff
@@ -450,6 +451,37 @@ async function _authorize({
   };
 }
 
+/**
+ * Gets an expectedRootCapability from an invocationTarget.
+ *
+ * @param {object} options - Options to use.
+ * @param {string | Array<string>} options.invocationTarget - An
```

Comment:
Should only ever be a string.

Reply:
Addressed https://github.com/digitalbazaar/bedrock-kms-http/pull/14/commits/8b5eb53e0fe3780dcea82786dab1571f7f0b083b